Amazon.QLDB.Driver.Tests/AsyncResultTests.cs

```diff
@@ -27,37 +27,37 @@ public class AsyncResultTests
     {
         private static AsyncResult result;
         private static Mock<Session> mockSession;
-        private static readonly MemoryStream memoryStream = new();
-        private static readonly ValueHolder valueHolder = new()
+        private static readonly MemoryStream MemoryStream = new();
```

Comment:
Any particular reason why we are capitalizing these variables?

Reply:
Because of styling warning. `readonly` values should start with capital letter